### PR TITLE
Feat/meet the team

### PR DESCRIPTION
**Pull request by Josh-Alhassan**
meet-the-team

Added meet the Team section

### CODE REVIEW

File: src/sections/MeetTheTeam/MeetTheTeam.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          <p className="meet-our-team__description">
+            Get to Know the Passionate Individuals Committed to Empowering Learners, Each Bringing Unique Skills and Perspectives to Drive Positive
+            Change in Communities Worldwide.
+          </p>
+        </div>
+
+        <div className="meet-our-team__carousel carousel">
+          <Slider {...settings}>
+            {/* ----------- */}
+            <div className="image-box box">
```
Comment (edustipend-root) [inline src/sections/MeetTheTeam/MeetTheTeam.jsx:66]: I would prefer we don't hardcode this instead we should have this read an array and render dynamically.

So something like

```
const teamMembers = [
  { name: 'Uduak', role: 'Founder', img: 'linkToImage'
]
```

And then you would render from the array dynamically.

**Review by edustipend-root — CHANGES_REQUESTED** [review]
@Josh-Alhassan, please review my comments.
Also, I didn't see any tests added to this PR.

File: src/sections/MeetTheTeam/MeetTheTeam.jsx (hunk 1 above)
Comment (Josh-Alhassan) [reply src/sections/MeetTheTeam/MeetTheTeam.jsx]: I refactored the code to read from an array and dynamically render.

File: src/assets/Blessing.png
Comment (edustipend-root) [inline src/assets/Blessing.png:1]: @Josh-Alhassan why do we have these assets here and also in the `team` folder? Similar to `Joshua.png` and `Uduak.png`.

What's the need for the duplicates?

**Review by edustipend-root — CHANGES_REQUESTED** [review]
@Josh-Alhassan, we have duplicate images which are not necessary. Please clean that up so I can get this merged in.

**Pull request closed without merging**